### PR TITLE
made a beautiful container

### PR DESCRIPTION
**Pull request by AshIgnYeo**
updated the styles for application.html.erb

### CODE REVIEW

**Comment by suansen** [comment]
beautiful container doesn't look that beautiful. Maybe add grid.

**Comment by AshIgnYeo** [comment]
@suansen oh yeh I forgot we were adding the grid 8 column. my bad.

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -16,7 +16,11 @@
     <%= render 'shared/navbar' %>
     <%= render 'shared/flashes' %>
     <div class="container">
-      <%= yield %>
+      <div class="row">
+        <div class="col-8 mx-auto">
```
Comment (AshIgnYeo) [inline app/views/layouts/application.html.erb:20]: @suansen hope you're happy now

**Review by suansen — APPROVED** [review]
(inner thoughts: I have no idea what I'm doing...)

looks good.

**Pull request merged**